### PR TITLE
add note about restarting IDE to multi-file schema docs

### LINKED ISSUE #6076: Add instructions for restarting IDE after prismaSchemaFolder enablement

### PR DESCRIPTION
**Pull request by jharrell**
Resolves #6076

### CODE REVIEW

File: content/200-orm/100-prisma-schema/10-overview/04-location.mdx [hunk 1]
```diff
@@ -82,6 +82,12 @@ datasource db {
 }
 ```
 
+:::info
+
+When first updating to the `prismaSchemaFolder` preview feature, your IDE may not recognize the new format immediately. Be sure to restart your IDE to confirm you have the latest version of the Prisma VS Code Extension
```
Comment (jharrell) [inline content/200-orm/100-prisma-schema/10-overview/04-location.mdx]:
```suggestion
When first updating to the `prismaSchemaFolder` preview feature, your IDE may not recognize the new format immediately. Be sure to restart your IDE to confirm you have the latest version of the Prisma VS Code Extension installed.
```

File: content/200-orm/100-prisma-schema/10-overview/04-location.mdx [hunk 2]
```diff
@@ -82,6 +82,12 @@ datasource db {
 }
 ```
 
+:::info
+
+When first updating to the `prismaSchemaFolder` preview feature, your IDE may not recognize the new format immediately. Be sure to restart your IDE to confirm you have the latest version of the Prisma VS Code Extension installed.
```
Comment (janpio) [inline content/200-orm/100-prisma-schema/10-overview/04-location.mdx]: What is this based on? That should not be the case. If this really happens, we should treat it as a bug and fix it.

Comment (jharrell) [reply content/200-orm/100-prisma-schema/10-overview/04-location.mdx]: Sorry, let me clarify
```suggestion
When first updating to Prisma ORM 5.15.0 or later, your IDE may not recognize the new multi-file format immediately. Be sure to restart your IDE to confirm you have the latest version of the Prisma VS Code Extension installed.
```

Comment (janpio) [reply content/200-orm/100-prisma-schema/10-overview/04-location.mdx]: Ah ok, that is less drastic.

Is restarting really enough? Not better "check your extensions that you have at least version 5.15.0 of the Prisma VS Code Extension isntalled" or something like that?

Comment (jharrell) [reply content/200-orm/100-prisma-schema/10-overview/04-location.mdx]: We already have instructions for updating the VSCode extension. This block was specifically for some confusion that folks in Discord had. So just covering more bases.

**Pull request merged**